src/xml2js.coffee

```diff
@@ -92,6 +106,7 @@ class exports.Builder
   buildObject: (rootObj) ->
     attrkey = @options.attrkey
     charkey = @options.charkey
+    cdata = @options.cdata
```

Comment:
I don't think this needs to be a local, I'd just use `@options.cdata` everywhere.